nemo_aligner/models/nlp/gpt/gpt_sft_model.py

```diff
@@ -32,6 +37,35 @@
 
 
 class GPTSFTModel(MegatronGPTModel, SupervisedInterface):
+    def __init__(self, cfg, trainer):
+        super().__init__(cfg, trainer)
+        inference_params = dict(cfg.get("inference", {}))
+        # note that this will fail is import path is not available when the model is restored
+        # this is by design as it might not be possible to use model correctly without a matching
+        # inference strategy
+        if "strategy" in inference_params:
+            inference_params["strategy"] = hydra.utils.instantiate(inference_params["strategy"], model=self)
```

Comment:
Can you add support for `strategy=null`? This may be useful to disable a strategy that already exists in the original model config.

Reply:
sure, let me do that